py/torch_tensorrt/dynamo/conversion/impl/elementwise/ops.py

```diff
@@ -239,3 +243,299 @@ def _add_layer(
         input_val = clamp_max_layer.get_output(0)
 
     return input_val
+
+
+def add(
+    network: TRTNetwork,
+    target: Target,
+    source_ir: Optional[SourceIR],
+    name: str,
+    lhs_val: Union[TRTTensor, int, float],
+    rhs_val: Union[TRTTensor, int, float],
+) -> TRTTensor:
+    if isinstance(lhs_val, TRTTensor):
+        lhs_val = trt_cast_int_to_float(network, name, lhs_val)
+
+    if isinstance(rhs_val, TRTTensor):
+        rhs_val = trt_cast_int_to_float(network, name, rhs_val)
```

Comment:
This can be removed, unless there is reason to cast the `add` operator to a float.